zk_toolbox/README.md

```diff
@@ -42,16 +42,18 @@ that connects all ZK chains, like the BridgeHub, the shared bridges, and state t
 To create a ZK Stack project, you must first create an ecosystem:
 
 ```bash
+# Skip this step if you are manually building from a local copy of the ZkSync repository (which is an existing ecosystem)
```

Comment:
It may not be the correct wording.

They could think that they don't need to create it anyway..
but they could create it somewhere outside zksync repository

Reply:
True that. Let me think about how to improve this.